db/re/item_combos.yml

```diff
@@ -46814,7 +46814,8 @@ Body:
          bonus2 bSkillAtk,"EM_ELEMENTAL_BUSTER_WIND",2*.@sum;
          bonus2 bSkillAtk,"EM_ELEMENTAL_BUSTER_POISON",2*.@sum;
          bonus2 bSkillAtk,"EM_ELEMENTAL_BUSTER_GROUND",2*.@sum;
-         if (getskilllv("EM_ELEMENTAL_BUSTER") > 0) {
+         .@class_ = geteleminfo(2);
+         if (getskilllv("EM_ELEMENTAL_BUSTER") > 0 && .@class_ >= 20816 && .@class_ <= 20820) {
```

Comment:
I'm surprised we don't export the constant for these. Should we go ahead an do that in this PR too?